apis/v1alpha2/httproute_types.go

```diff
@@ -732,6 +732,9 @@ type HTTPBackendRef struct {
 	// In either error case, the Message of the `ResolvedRefs` Condition
 	// should be used to provide more detail about the problem.
 	//
+	// Note that SSL passthrough is not supported for these backend refs, and
+	// if you're looking to implement that kind of functionality use TLSRoute.
+	//
```

Comment:
I think instead of adding this here, it might be better to add this to the Passthrough godocs here: https://github.com/kubernetes-sigs/gateway-api/blob/master/apis/v1alpha2/gateway_types.go#L337-L340

Reply:
Moved as suggested in 34d584e165ca992aa17de3f68b3a331d93ee9eaa :+1: